src/i18n/Translate/Translate.Debug.js

```diff
@@ -0,0 +1,164 @@
+import PropTypes from "prop-types"
+import styled from "styled-components"
+// import i18next from "i18next"
+// import { i18nLogger, useI18n } from "../I18n"
+// import { getComponent, handleArray } from "../utils"
+import React, { useState } from "react"
+
+/**
+ * Jsx Component for translating
+ * @param children
+ * @param i18n
+ * @param data
+ * @returns {null|*}
+ * @constructor
+ */
+const getValue = (i18n, list) => {
+  for (const [key, value] of Object.entries(list)) {
+    if (key === i18n) {
+      return value
+    }
+  }
+}
+
+const Translate = ({ children, value, i18n, allLangs }) => {
```

Comment:
Change name to TranslateDebug

Reply:
Fikset

Reply:
Helt fin endring, bare husk at filen kan godt hete `Translate.Debug`, men funksjonen inni er det viktig at vi er flinke å navngi fordi Component.displayname er basert på det.